src/sentry/features/__init__.py

```diff
@@ -132,6 +132,7 @@
 default_manager.add("organizations:release-adoption-stage", OrganizationFeature, True)
 default_manager.add("organizations:release-archives", OrganizationFeature)
 default_manager.add("organizations:release-comparison", OrganizationFeature, True)
+default_manager.add("organizations:release-comparison-performance", OrganizationFeature, True)
```

Comment:
Should this be also added here?
https://github.com/getsentry/sentry/blob/8af4bd9cf6d2cea03a84e6c381aff7b259246821/src/sentry/conf/server.py